ci/storage/test_main_storage_prepare_node.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
In general, @saurabhtandan is working on some scripts for testing clusters before DAOS installations. It would probably be a good idea to (future) formally integrate those scripts alongside this one.

Reply:
> I also would like tests that can be used for to validate a new cluster or a new snapshot image to be able to to be optionally run.

The scripts @saurabhtandan is working on aim to do this for the upcoming DAOS installations. Maybe once those are flushed out more we can figure out proper integration into CI